### PR TITLE
chore(release): 3.17.0

### PR DESCRIPTION
**Pull request by sfoster1**
This is release 3.17.0.

### CODE REVIEW

File: app-shell/build/release-notes.md [hunk 1]
```diff
@@ -1,10 +1,24 @@
...
+  labware calibration list
+  ([#5147](https://github.com/opentrons/opentrons/issues/5147))
+- Fix an issue where custom labware files or Protocol Designer protocols with
+  an all-caps ``.JSON`` exception would not be handled properly
+  ([#5151](https://github.com/opentrons/opentrons/issues/5151))
+- Because module commands now respect protocol pause, module interaction via the
+  Opentrons App is now disabled when the protocol is paused.
+  
+## New Features
+- OT-2s that are on at least version 3.17.0 and connected via USB can now be
```
Comment (mcous) [inline app-shell/build/release-notes.md]: No Wi-Fi actions (including disconnect) are currently gated behind being connected over the wired connection. It's definitely our recommendation, but the UI does not currently warn nor stop you if you're only connected via Wi-Fi

See #2551

**Review by nusrat813 — APPROVED** [review]
👍

**Review by Kadee80 — APPROVED** [review]
🏭

File: app-shell/build/release-notes.md (hunk 1 above)
Comment (sfoster1) [reply app-shell/build/release-notes.md]: OK. Let's leave it in for now I guess

**Pull request merged**